### ISSUE TITLE
Navigation Banner blank

### ISSUE THREAD

**Issue by mqln** [mapbox/mapbox-navigation-ios#1995]
**0.29.1**

The top banner is not populating, at least most of the time. Sometimes it'll populate after what seems to be a large enough location update, but I'm unsure.

![image](https://user-images.githubusercontent.com/1429466/53522144-6a59ec80-3a9f-11e9-8150-6a6a0d598345.png)

I'm getting the rout via:

```
        let origin = Waypoint(coordinate: CLLocationCoordinate2D(latitude: userLocation.coordinate.latitude, longitude: userLocation.coordinate.longitude), name: "Mapbox")
        let destination = Waypoint(coordinate: CLLocationCoordinate2D(latitude: end.latitude, longitude: end.longitude), name: "White House")
        
        let options = NavigationRouteOptions(waypoints: [origin, destination])
        Directions.shared.calculate(options) { (waypoints, routes, error) in
            guard let routes = routes else { return }
```
... and presenting the nav via:

```
guard let route = routes?.first else { return }
navigationViewController = NavigationViewController(for: route)
present(navigationViewController, animated: true, completion: nil)
```

This was working smoothly for me about a year ago, when I was using **0.13**

Cheers!

**Comment by 1ec5** [comment]
This looks similar to #1972, which was fixed in #1983 in v0.30.0. However, that issue only affected LegacyRouteController, which you have to opt into explicitly. From the code above, it looks like the default RouteController is being used.

**Comment by mqln** [comment]
Great temp fix, cheers @1ec5 .
